packages/camera/README.md

```diff
@@ -0,0 +1,105 @@
+# Camera Plugin
+
+[![pub package](https://img.shields.io/pub/v/camera.svg)](https://pub.dartlang.org/packages/camera)
+
+A Flutter plugin for iOS and Android allowing access to the device cameras.
+
+## Features:
+
+* Display live camera preview in a widget.
+* Snapshots can be captured and saved to a file.
+
+## Installation
+
+First, add `camera` as a [dependency in your pubspec.yaml file](https://flutter.io/using-packages/).
+
+### iOS
+
+Add a row to the `ios/Runner/Info.plist` of your app with the key `Privacy - Camera Usage Description` and a usage description.
+
+Or in text format add the key:
+
+```xml
+	<key>NSCameraUsageDescription</key>
+    <string>Can I use the camera please?</string>
+```
+
+### Android
+
+#### Permissions
+Add the right permissions and features in your Android Manifest file, located in `<project root>/android/app/src/main/AndroidManifest.xml:
+
+For example:
+
+```xml
+<uses-permission android:name="android.permission.CAMERA"/>
```

Comment:
Does this has to be specified also in the app? It is already specified in the plugin.

Reply:
No - actually the app needs to specify no extra permissions!